[2024-05-25T23:55:25.694+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-25T23:55:25.717+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DAG_catedra_airflow.first-step-el scheduled__2024-05-25T23:29:00+00:00 [queued]>
[2024-05-25T23:55:25.725+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DAG_catedra_airflow.first-step-el scheduled__2024-05-25T23:29:00+00:00 [queued]>
[2024-05-25T23:55:25.726+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-25T23:55:25.738+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): first-step-el> on 2024-05-25 23:29:00+00:00
[2024-05-25T23:55:25.744+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=3011) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-25T23:55:25.746+0000] {standard_task_runner.py:63} INFO - Started process 3013 to run task
[2024-05-25T23:55:25.746+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'DAG_catedra_***', 'first-step-el', 'scheduled__2024-05-25T23:29:00+00:00', '--job-id', '466', '--raw', '--subdir', 'DAGS_FOLDER/dag_clase.py', '--cfg-path', '/tmp/tmp3_9v_b8g']
[2024-05-25T23:55:25.747+0000] {standard_task_runner.py:91} INFO - Job 466: Subtask first-step-el
[2024-05-25T23:55:25.788+0000] {task_command.py:426} INFO - Running <TaskInstance: DAG_catedra_airflow.first-step-el scheduled__2024-05-25T23:29:00+00:00 [running]> on host b2cfee8f0b2b
[2024-05-25T23:55:25.876+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='DAG_catedra_***' AIRFLOW_CTX_TASK_ID='first-step-el' AIRFLOW_CTX_EXECUTION_DATE='2024-05-25T23:29:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-25T23:29:00+00:00'
[2024-05-25T23:55:25.878+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-25T23:55:25.883+0000] {logging_mixin.py:188} INFO - Connection to database was succesful
[2024-05-25T23:55:25.893+0000] {logging_mixin.py:188} INFO - Query completed
[2024-05-25T23:55:25.895+0000] {logging_mixin.py:188} INFO - execute_query_result_set()
[2024-05-25T23:55:25.900+0000] {logging_mixin.py:188} INFO - Connection to database was succesful
[2024-05-25T23:55:25.906+0000] {logging_mixin.py:188} INFO - Inserts performed wihout fail
[2024-05-25T23:55:25.907+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-25T23:55:25.907+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-25T23:55:25.914+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=DAG_catedra_***, task_id=first-step-el, run_id=scheduled__2024-05-25T23:29:00+00:00, execution_date=20240525T232900, start_date=20240525T235525, end_date=20240525T235525
[2024-05-25T23:55:25.960+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-25T23:55:25.995+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-25T23:55:25.999+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
